deploy-vip.sh

```diff
@@ -27,16 +27,14 @@ if [[ "$1" == "pr" || "$1" == "-pr" ]]; then
 	PR=true
     VIP_PR_BRANCH="merge/"
 
-    # Append Jira ticket in merge branch name if present
-    JIRA_TICKET="$( echo $BUDDY_EXECUTION_REVISION_MESSAGE | grep -oE -m 1 "[A-Za-z]+-[0-9]+" | head -1 | tr ' ' '-' | xargs echo -n )"
-
-    if [ -n $JIRA_TICKET ]; then
-        VIP_PR_BRANCH+="${JIRA_TICKET}/"
-    fi
+    # TODO Append Jira ticket in merge branch name if present. The logic below was creating broken branch names and is worth further testing.
```

Comment:
I wonder if VIP uses this or may depend on its presence... perhaps @mslinnea or @srtfisher  can confirm? 

I'm curious if you have an example of broken branch names? The regex seems to match jira-like things... and convert/trim spaces appropriately, but if this is important to teams, we should consider retaining it?

Reply:
we do not use this script on NYP but do something similar:

- https://github.com/alleyinteractive/nypost-go/blob/production/.buddy/Deploy_Production_to_VIP_Master.yml#LL32C15-L32C36
- https://github.com/alleyinteractive/nypost-go/blob/production/.buddy/Deploy_Production_to_VIP_Master.yml#L69-L81